pkg/app/app.go

```diff
@@ -33,9 +33,6 @@ func CreateApp() App {
 
 	// Writing directly to stderr because log has not been bootstrapped
 	if app.args, err = CreateArgs(); err != nil {
-		os.Stderr.WriteString("ERROR: Failed to validate input\n")
-		os.Stderr.WriteString(err.Error() + "\n")
-		ArgsUsage()
 		os.Exit(127)
```

Comment:
I would also worry about returning 127 for the help option. I would think that we would want the more general 1 at least.